Makefile

```diff
@@ -20,7 +20,6 @@ test: lint generate
 install-tools:
 	@mkdir -p bin
 	go install github.com/kisielk/errcheck
-	go install honnef.co/go/tools/cmd/staticcheck
```

Comment:
I opted to remove `staticcheck` because I was having trouble getting it to install across all versions of go.